x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/__test__/ServiceOverview.test.tsx

```diff
@@ -11,7 +11,7 @@ import { toastNotifications } from 'ui/notify';
 import * as callApmApi from '../../../../services/rest/callApmApi';
 import { ServiceOverview } from '..';
 import * as urlParamsHooks from '../../../../hooks/useUrlParams';
-import * as coreHooks from '../../../../hooks/useCore';
+import * as kibanaCore from '../../../../../../observability/public/context/kibana_core';
```

Comment:
FWIW, we only export things from `observability/public` by way of Kibana convention, but if you try to import from there and then use `jest.spyOn()`, it blows up because the re-exported stuff in the `public/index` file doesn't mesh with `jest.spyOn`. 

For more info, see: https://stackoverflow.com/a/53307822